number/src/lib.rs

```diff
@@ -31,6 +33,60 @@ pub fn log2_exact(n: BigUint) -> Option<usize> {
         .filter(|zeros| n == (BigUint::from(1u32) << zeros))
 }
 
+pub type Columns<F> = Vec<(String, Vec<F>)>;
```

Comment:
I think this type name over-simplifies (i.e. there should be no typedef ;) ). I would like to know that this is a vector and that it contains the name as a string.

Reply:
done